Makefile

```diff
@@ -8,8 +8,7 @@ clean:
 		rm -fr *.egg-info
 
 install:
-		pip install -e .
-		pip install -r dev_requirements.txt
+		pip install -e .[test]
```

Comment:
Those are [requirement specifiers](http://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-dependencies), whose syntax, in this case, is described in [PEP 508](https://www.python.org/dev/peps/pep-0508/) as  `extras` and are comma-delimited ids within square brackets which pip interprets case-insensitively and maps to entries in `setup.py`'s `extras_require`. Quotes would be superfluous.